addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -247,10 +247,19 @@
 				<aspectratio aligny="center">keep</aspectratio>
 				<texture>$INFO[Player.Art(thumb)]</texture>
 			</control>
+			<control type="image">
+				<right>30</right>
+				<top>20</top>
+				<width>200</width>
+				<height>200</height>
+				<aspectratio aligny="center">keep</aspectratio>
+				<texture>$INFO[PVR.EpgEventIcon]</texture>
+				<visible>VideoPlayer.HasEpg + !String.IsEmpty(PVR.EpgEventIcon)</visible>
```

Comment:
Shouldn't we add a `fallback="DefaultTVShows.png"` to the texture tag? Otherwise we will have an ugly empty space at the right if no epg image is avalaible.  Then, the visible condition can be reduced to `<visible>VideoPlayer.HasEpg</visible>`

Reply:
I think in this case the empty space is better than a TV icon that is not related with anything in the bar. I would prefer an empty space.

Reply:
@ksooo Here are the screens.
Empty space:
![image](https://user-images.githubusercontent.com/196953/44909514-96c2c500-ad17-11e8-87c7-9d88bb54de5f.png)
TV Icon:
![image](https://user-images.githubusercontent.com/196953/44909539-afcb7600-ad17-11e8-9b99-f28e945d6c64.png)
Empty space all the way for me.

Reply:
You need a longer, multi-line plot to see what I mean. Then it makes a difference,

Reply:
IFAIK, in Kodi we always use fallback images if no thumbnail is avail.

Reply:
Best case imo is to make the plot text field wider if no epg event icon is avail. That would be my number 1 candidate.

Reply:
I and tested it and saw what you mean by empty square. Made the changes.
I tried to do something like:
```
<variable name="VideoPlayerPlotRight">
	<value condition="VideoPlayer.HasEpg">260</value>
	<value>30</value>
</variable>
```
```
<right>$VAR[VideoPlayerPlotRight]</right>
```
But didn't work, still don't understand when and where can we use $VAR or not.